src/DateTimeFactory.php

```diff
@@ -1,18 +1,27 @@
 <?php
+declare(strict_types = 1);
 
 namespace Damejidlo\DateTimeFactory;
 
-use DateTime;
+use Nette\SmartObject;
+use Nette\Utils\DateTime;
```

Comment:
Btw, why? I dont see any added value from it.

Reply:
Because removal was a fucking huge BC break. Nette dependency can/will be removed together with deprecated DateTimeFactory in the future. If you don't want Nette, you can stay on v1.1.